docs/pages/api/rbac.mdx

```diff
@@ -0,0 +1,984 @@
+version: v5
+metadata:
+  name: sync-kubernetes-rbac-impersonator
+spec:
+  allow:
+    impersonate:
+      roles:
+      - sync-kubernetes-rbac
+      users:
+      - sync-kubernetes-rbac
+```
+
+Create the `sync-kubernetes-rbac-impersonator` role: 
+
+```code
+$ tctl create -f sync-kubernetes-rbac-impersonator.yaml
+```
+
+(!docs/pages/includes/add-role-to-user.mdx role="sync-kubernetes-rbac-impersonator"!)
+
+You will now be able to generate signed certificates for the `sync-kubernetes-rbac`
+role and user.
+
+### Install the Teleport Kubernetes Service
+
+We will enable your client application to communicate with your Kubernetes
+cluster via the Teleport Kubernetes Service, which forwards requests after
+authorizing them. While this step is not strictly necessary with a local
+`minikube` cluster, it demonstrates one way to use Teleport to securely access
+your external RBAC system's API.
+
+(!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
+
+Request a token that the Kubernetes Service will use to join your Teleport
+cluster:
+
+```code
+$ tctl nodes add --roles=kube --ttl=10000h --format=json
+```
+
+Copy this token so you can use it when running the Teleport Kubernetes Service.
+
+Ensure that you are connected to the right Kubernetes cluster:
+
+```code
+$ kubectl config use-context minikube
+Switched to context "minikube".
+```
+
+Install the Teleport Kubernetes Service in your cluster, assigning <Var
+name="proxy-address" /> to the host **and port** of your Teleport Proxy Service
+(e.g., `mytenant.teleport.sh:443`) and <Var name="token" /> to the token you
+requested earlier:
+
+```code
+$ helm install teleport-agent teleport/teleport-kube-agent \
+  --set kubeClusterName=minikube \
+  --set proxyAddr=<Var name="proxy-address" /> \
+  --set authToken=<Var name="token" /> \
+  --create-namespace \
+  --namespace=teleport \
+  --set labels.environment=demo \
+  --version (=teleport.version=) 
+```
+
+After a few seconds, verify that you have deployed the Teleport Kubernetes
+Service by running the following command:
+
+```code
+$ kubectl -n teleport get pods
+```
+
+This should show that the Kubernetes Service is running:
+
+```text
+NAME               READY   STATUS    RESTARTS   AGE
+teleport-agent-0   1/1     Running   0          22s
+```
+
+`tsh` should indicate that the cluster has registered with Teleport:
+
+```code
+$ tsh kube ls
+Kube Cluster Name Labels           Selected
+----------------- ---------------- --------
+minikube          environment=demo
+```
+
+## Step 3/4. Write the client application
+
+At this point, we have set up an external RBAC system to use for generating
+Teleport roles and configured Teleport to allow our API client to interact with
+our Kubernetes cluster and Teleport cluster. In this step, we will write our
+client application.
+
+### Set up your Go project
+
+Download the source code for the API client application:
+
+```code
+$ git clone https://github.com/gravitational/teleport
```

Comment:
Thanks! I'll use the second command since it's a bit more straightforward than the third, and the performance difference in this context between option 2 and option 3 is pretty small.